python/fbprophet/plot.py

```diff
@@ -44,13 +45,14 @@ def plot(
         in the figure, if available.
     xlabel: Optional label name on X-axis
     ylabel: Optional label name on Y-axis
+    figsize: Optional tuple width, height in inches.
 
     Returns
     -------
     A matplotlib figure.
     """
     if ax is None:
-        fig = plt.figure(facecolor='w', figsize=(10, 6))
+        fig = plt.figure(facecolor='w', figsize)
```

Comment:
FYI, this line introduced a bug that disallows the entire `fbprophet` module from being imported.
`SyntaxError: positional argument follows keyword argument`


Should be:
```python
fig = plt.figure(facecolor='w', figsize=figsize)
```